### ISSUE TITLE
Clean up sysTools

### ISSUE THREAD

**Issue by astrobokonon** [LowellObservatory/DockerDev#2]
Right now, the [sysTools](https://github.com/LowellObservatory/DockerDev/tree/master/sysTools) docker-compose setup is a bit screwy.  If you ```docker-compose up -d``` in that base directory, everything but jekyll will come up; jekyll has to be started in the *mainLandingPage* directory rather than the root, but if you do a ```docker-compose up -d``` there than nginx will barf.

Right now the proper sequence is:

- In the base directory, ```docker-compose up -d omsa82 nginx```

- Navigate into the mainLandingPage directory, then ```docker-compose up -d jekyll```

Also, nginx is starting up with a random name and I don't know why. This all needs to be sorted out.

**Comment by astrobokonon** [comment]
Might just be able to adjust the volume mount point of the jekyll service like this? 

Change this:
```
jekyll:
    container_name: jekyll
    command: jekyll serve
    image: jekyll/jekyll:latest
    volumes:
      - $PWD:/srv/jekyll
      - $PWD/vendor/bundle:/usr/local/bundle
    ports:
     - 4000:4000
     #- 35729:35729
     #- 6000:3000
    network_mode: "host"
```
to this:
```
jekyll:
    container_name: jekyll
    command: jekyll serve
    image: jekyll/jekyll:latest
    volumes:
      - $PWD/mainLandingPage/:/srv/jekyll
      - $PWD/vendor/bundle:/usr/local/bundle
    ports:
     - 4000:4000
     #- 35729:35729
     #- 6000:3000
    network_mode: "host"
```

**Comment by astrobokonon** [comment]
Yep, the above did at least the jekyll portion of this. Looks like fixing this also fixed the nginx name confusion, it must have been periodically confused when I screwed up and did ```docker-compose up -d``` from the *mainLandingPage* directory; it would make sense to be confused there since nginx wouldn't know where to find some of its environment variables since they were one directory down at that point.

**Comment by astrobokonon** [comment]
Closed by 2c3ca2fae25a0d6f23c6b9406b25b63c5d1d4023
